### DISCUSSION TITLE
Synthesis Failed, no bitfile found

### DISCUSSION THREAD

**Discussion by aidanrhind** [Xilinx/finn#994] [Q&A] [answered]
Hello all, I'm fairly new to finn and am having an error during synthesis with my model. I'm attempting to create a qlenet5 accelerator for a kv260 using the dev branch, Xilinx tools = 2022.1, and ubuntu 20.04 in vbox. Model was built and trained with brevitas and converted to onnx for finn export. I've attached the vivado synthesis
output log and the finn-onnx model I am trying to convert to hls. For some reason synthesis keeps failing for both the Zynqbuild with manually applied streamlining and folding as well as the standard finn dataflowbuild methods. I've been following some of the e2e example notebooks and am not sure why this is the case. I saw a similar error in another discussion post and maxed the amount of ram for my vm as it said that should fix the issue but unfortunately it persists for me. Any help would be much appreciated!

![Screenshot from 2024-02-25 11-56-08](https://github.com/Xilinx/finn/assets/100760453/a006fd1c-76bd-477e-95ec-91bb108819da)

![lenet5_quant_dataflow onnx](https://github.com/Xilinx/finn/assets/100760453/1fb36540-0f70-485e-af6c-939bb5334a75)

[vivado.log](https://github.com/Xilinx/finn/files/14397731/vivado.log)

**Comment by fpjentzsch** [comment]
Hi, the Vivado log indicates that the run for one of the IPs has failed:
WARNING: [Vivado 12-8222] Failed run(s) : 'top_StreamingDataflowPartition_1_0_synth_1'

You'll have to dig deeper into the logs of that run to find the cause.

**Reply by aidanrhind** [reply, marked as answer]
Resolved this by checking the top_StreamingDataflowPartition_1_0_synth_1 log and the model was too large for the kv260. I readjusted the folding parameters and got an out of memory error. I ended up upgrading the RAM on my laptop from 16gb to 48gb, and assigning my vbox vm over 32gb of ram and I finally was able to get the model to synthesize and generate the pynq driver. Moral of the story for me: have more than 16gb of RAM and check the logs to see if the model fits device.
